packages/node/src/rest/types/organizations/index.ts

```diff
@@ -45,14 +45,22 @@ export enum OrganizationSocialType {
 export type OrganizationMember = {
   user: User;
   role: OrganizationMemberRole;
+  status: OrganizationMemberStatus;
```

Comment:
Donc si un membre est pas accepté, il est quand même dans la liste mais avec le status "rejected" ?

Reply:
So when we search our member list, we also see those who have rejected the invitation?

Reply:
We could use this to suggest that Admins invite the person again.

Reply:
je changerai pas ton pdv